src/nfts/NftsInfoCarousel.tsx

```diff
@@ -0,0 +1,350 @@
+        <ScrollView
+          showsHorizontalScrollIndicator={false}
+          horizontal={true}
+          contentContainerStyle={styles.carouselScrollViewContentContainer}
+          style={styles.nftImageCarousel}
+          testID="NftsInfoCarousel/NftImageCarousel"
+        >
+          {nfts.map((nft) => {
+            return (
+              <View
+                key={`${nft.contractAddress}-${nft.tokenId}`}
+                style={styles.nftThumbnailSharedContainer}
+              >
+                <NftThumbnail
+                  onPress={handleOnPress(nft)}
+                  // Use contractAddress and tokenId for a unique key
+                  isActive={
+                    `${activeNft.contractAddress}-${activeNft.tokenId}` ===
+                    `${nft.contractAddress}-${nft.tokenId}`
+                  }
+                  // If there's no nft metadata, show a red info icon instead
+                  error={!nft.metadata}
+                  nft={nft}
+                />
+              </View>
+            )
+          })}
+        </ScrollView>
+      </View>
+    )
+  }
+
+  // Display error screen no Nfts provided
+  if (nfts.length === 0) {
+    return <NftsLoadErrorScreen />
+  }
+
+  return (
+    <SafeAreaView edges={['top']} style={styles.safeAreaView} testID="NftsInfoCarousel">
+      <ScrollView>
+        {activeNft.metadata && (
+          <FastImage
+            testID={`NftsInfoCarousel/NftImage-${activeNft.contractAddress}-${activeNft.tokenId}`}
+            style={[
+              { height: scaledHeight },
+              styles.mainImage,
+              // Put a border radius on the image when loading to match placeholder
+              isLoading && styles.borderRadius,
+            ]}
+            source={{
+              uri: activeNft.media.find((media) => media.raw === activeNft.metadata?.image)
+                ?.gateway,
+            }}
+            onLoad={(e) => {
+              setScaledHeight(
+                scaleImageHeight(e.nativeEvent.width, e.nativeEvent.height, variables.width)
+              )
+            }}
+            onLoadEnd={() => setIsLoading(false)}
+            onError={() =>
+              Logger.error(
+                TAG,
+                `Error loading Nft image for Nft contractAddress: ${activeNft.contractAddress} tokenId: ${activeNft.tokenId}`
+              )
+            }
+            resizeMode={FastImage.resizeMode.contain}
+          >
+            {isLoading && <MainImagePlaceholder />}
+          </FastImage>
+        )}
+        {/* This could happen if the indexer is experiencing issues with particular nfts */}
+        {!activeNft.metadata && (
+          <View style={styles.nftImageLoadingErrorContainer}>
+            <Text style={{ color: colors.light }}>{t('nftInfoCarousel.nftImageLoadError')}</Text>
+          </View>
+        )}
+
+        {nfts.length > 1 && <NftImageCarousel />}
+        <View style={styles.sectionContainer}>
```

Comment:
do you think a structure like this is easier to read?
```
{activeNft.metadata && (
  <>
    {activeNft.metadata.title}
    {activeNft.metadata.description}
    {activeNft.metadata.attributes && CodeToRenderAttributes}
  </>
)}
```
the current structure with all the ternaries makes it a little hard to understand which elements are likely to be there or missing, since everything looks optional